agent/roles/manifests/templates/agent-cluster-install_yaml.j2

```diff
@@ -43,7 +43,7 @@ spec:
     - cidr: {{ external_subnet_v6 }}
 {% endif %}
     networkType: {{ network_type }}
-{% if platform_type == "none" %}
+{% if (platform_type == "none") or (platform_type == "external") %}
```

Comment:
In addition, now that the assisted-service PR to add external type merged, ExternalPlatformSpec should be added here. https://github.com/openshift/assisted-service/pull/5438/files#diff-8a5d8069ae1e2777920acfce8fd2d122f8e62c3e89b9c20b791b8cd35776dc9eR198

Reply:
Fixed